src/components/Calculator.js

```diff
@@ -1,55 +1,49 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/Calculate';
 
-class Calculator extends Component {
-  constructor(propbs) {
-    super(propbs);
-    this.state = {
-      value: null,
-      next: null,
-      operation: null,
-    };
-    this.handleClick = this.handleClick.bind(this);
-  }
+export default function Calculator() {
+  const initState = {
```

Comment:
- [ ] [OPTIONAL] Please consider using ES6 arrow function syntax as it is important to start adopting the newest changes and improvements in the JavaScript language in order to write better code 👍

Reply:
Hi @codepantha,
Thanks for your suggestion. I hope you will find the changes in the next commit.

Reply:
Hi @codepantha,
I changed the repository according to your suggestion. Would you please suggest to me what can I do? Merge the new branch or you want to review.

Thanks.